ic-helpers/src/tokens.rs

```diff
@@ -238,14 +241,19 @@ impl Tokens256 {
     /// Number of bytes needed to represent the value.
     pub const BYTE_LENGTH: usize = 256 / 8;
 
-    /// Converts the value to Tokens128. Returns `None` if the value is greater than `Tokens128::MAX`.
+    /// Converts the value to Tokens128. Returns `None` if the value is greater
+    /// than `Tokens128::MAX`.
     pub fn to_tokens128(&self) -> Option<Tokens128> {
```

Comment:
What issue does this fix? Could you please a test that reproduces the existing bug and shows this fixes it?

Reply:
Hey, Francesco. There's already test that fails before this fix, but only in x32 environment. Unfortunately, it's not that trivial to run the tests with `wasm32` target in our CI, so I suggest to deal with that issue in a separate PR.

Reply:
@Maximkaaa we don't need to run the tests in wasm32, to reproduce the bug it is enough to execute them with  `--target=i686-unknown-linux-gnu`